specs/rollup-node.md

```diff
@@ -126,9 +109,16 @@ The rollup reads the following data from the [sequencing window][g-sequencing-wi
     - Batches not matching filter criteria are ignored:
       - `batch.epoch == sequencing_window.epoch`, i.e. for this sequencing window
       - `(batch.timestamp - genesis_l2_timestamp) % block_time == 0`, i.e. timestamp is aligned
-      - `min_l2_timestamp < batch.timestamp < max_l2_timestamp`, i.e. timestamp is within range
+      - `min_l2_timestamp <= batch.timestamp < max_l2_timestamp`, i.e. timestamp is within range
```

Comment:
Norswap suggested this as an AND condition here (& I agree): `min_l2_timestamp >= l1_timestamp`

Reply:
`min_l2_timestamp` is defined below, not something we can handle like a user input here in the filter condition `AND` list. And it always has to be `prev_l2_timestamp + l2_block_time`, or else there would be a gap.

Do you mean to define `min_l2_timestamp >= l1_timestamp` as an invariant, that just always holds true based on the other conditions?